letsencrypt/data/run.sh

```diff
@@ -123,6 +123,7 @@ else
         --preferred-challenges "$CHALLENGE" "${DOMAIN_ARR[@]}" --standalone
 fi
 
-# copy certs to store
-cp "$CERT_DIR"/live/*/privkey.pem "/ssl/$KEYFILE"
-cp "$CERT_DIR"/live/*/fullchain.pem "/ssl/$CERTFILE"
+# Get the last modified cert directory and copy the cert and private key to store
+CERT_DIR=$(ls -td $CERT_DIR/live/* | head -1)
```

Comment:
In the event there's a directory for a certificate received for a given domain (e.g. hass.example.com) and the configuration is updated to add additional certificates to that domain (e.g. homeassistant.example.com), Let's Encrypt appends `-0001`, `-0002`, etc to the end of the directory to ensure previously issued certificates are not overwritten. We simply need to make sure we're using the the most recently modified directory, as it'll be the one certbot just put the files in.